infrastructure/aws/staging/ecs-task-definition-task-processor.json

```diff
@@ -114,6 +114,22 @@
                 {
                     "name": "TASK_DELETE_BATCH_SIZE",
                     "value": "100000"
+                },
+                {
+                    "name": "INFLUXDB_ORG",
+                    "value": "ben.rometsch@bullet-train.io"
+                },
+                {
+                    "name": "INFLUXDB_URL",
+                    "value": "https://eu-central-1-1.aws.cloud2.influxdata.com"
+                },
+                {
+                    "name": "SSE_INFLUXDB_BUCKET",
+                    "value": "sse_dev"
```

Comment:
This is kinda confusing, can we use sse_staging?

Reply:
Done